docs/docs/modules/index.mdx

```diff
@@ -4,16 +4,18 @@ sidebar_class_name: hidden
 
 # Modules
 
-LangChain provides standard, extendable interfaces and external integrations for the following modules, listed from least to most complex:
+LangChain provides standard, extendable interfaces and external integrations for the following main modules:
 
 #### [Model I/O](/docs/modules/model_io/)
 Interface with language models
 #### [Retrieval](/docs/modules/data_connection/)
 Interface with application-specific data
-#### [Chains](/docs/modules/chains/)
-Construct sequences of calls
 #### [Agents](/docs/modules/agents/)
 Let chains choose which tools to use given high-level directives
+
+## Additional
+#### [Chains](/docs/modules/chains/)
```

Comment:
@baskaryan It is radical. Moving `Chains` into the second level of Docs. Do we still call it the `LangCHAIN`? ;)